.travis.yml

```diff
@@ -1,13 +1,69 @@
-sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: make test
+# Two types of tests:
+#
+# 1. shell tests run using bats
+# 2. build tests also run using bats
+#
+# Shell tests are run unconditionally on Linux (see jobs.include).
+#
+# Build tests can be skipped by starting the commit message with
+# '[skip build]'.
+#
+# Build tests are run on platforms generated from all combinations
+# of keys below.
 language: c
+
+os:
+- linux
+- osx
+
+dist:
+- xenial
+
+# https://docs.travis-ci.com/user/reference/osx/#macos-version
+osx_image:
+- xcode9.4
+- xcode10
+
 env:
-  - PYENV_NATIVE_EXT=
-  - PYENV_NATIVE_EXT=1
+- PYTHON_BUILD_VERSION=3.8-dev
+- PYTHON_BUILD_VERSION=3.7.2
+
+before_install:
+- date +%Y-%m-%dT%H:%M:%S
+
+install: git clone --depth 1 https://github.com/bats-core/bats-core.git bats
+
+# Default for auto-generated jobs.
+script: make test-build
+
+after_script: |
+  cat "$(ls -tr ${TMPDIR:-/tmp}/python-build.*.log | tail -1)"
```

Comment:
If what you are trying to do here is to print out build log, I think passing "--verbose" to python-build would be more _reliable_ than this hack 🤔

Reply:
I would have liked to specify the log file explicitly via e.g. environment variable instead of getting the latest `python-build.*.log` but we don't support that in `python-build` (yet).

No matter what we do it should be in the `after_script` step so it is in its own section separate from the test results/output (see [here](https://travis-ci.com/chrahunt/pyenv/jobs/178677022)).

Do you think it would be better to run with `--verbose` and echo `$output` to a specific file that we cat here instead of this?

Reply:
Ah, I guess I wasn't understanding what's going on there 😅 Since `python-build` is invoked via `bats`, to see the output `$output` has to be printed.... Okay, now everything is clear (to me) and I'm totally okay with current configurations. Thanks for explanation!